airflow/dag_processing/manager.py

```diff
@@ -219,12 +223,10 @@ def run(self):
         #     "Checking for new files in %s every %s seconds", self._dag_directory, self.dag_dir_list_interval
         # )
 
-        from airflow.dag_processing.bundles.manager import DagBundlesManager
-
-        DagBundlesManager().sync_bundles_to_db()
+        self._bundles_manager = DagBundlesManager().sync_bundles_to_db()
```

Comment:
```suggestion
        self._bundles_manager.sync_bundles_to_db()
```

Doesn't the factory create it for us?